thirdparty/mbedtls/patches/windows-arm64-hardclock.diff

```diff
@@ -0,0 +1,16 @@
+diff --git a/thirdparty/mbedtls/library/timing.c b/thirdparty/mbedtls/library/timing.c
+index 6c14a4fd01..151292e325 100644
+--- a/thirdparty/mbedtls/library/timing.c
++++ b/thirdparty/mbedtls/library/timing.c
+@@ -190,8 +190,10 @@ unsigned long mbedtls_timing_hardclock( void )
+ #endif /* !HAVE_HARDCLOCK && MBEDTLS_HAVE_ASM &&
+           __GNUC__ && __ia64__ */
+ 
+-#if !defined(HAVE_HARDCLOCK) && defined(_MSC_VER) && \
++// -- GODOT start --
++#if !defined(HAVE_HARDCLOCK) && defined(_WIN32) && \
```

Comment:
@bruvzg We need to PR that change upstream so we don't need to keep applying a patch here.

I'm not sure this would be the right/sufficient fix but if not, opening an issue upstream with the build error you encounter on MSVC arm64 would be good.

Reply:
Should be already fixed by https://github.com/Mbed-TLS/mbedtls/pull/5708

Reply:
That's weird because that PR was backported to 2.28 in April, so it should have been included in our build at the time you wrote this patch.